piety-promoted-v1/modernized/02-piety-promoted-v1.adoc

```diff
@@ -602,18 +634,23 @@ in an open prison, till the First month, 1661,
 when he was brought to the court at Boston with his chains and log at his heels.
 The court told him that he was found guilty, and was to die.
 
+[.discourse-part]
 William Leddra asked what evil he had done.
 
+[.discourse-part]
 The court replied that his own confession was as good as a thousand witnesses.
 
-William Leddra: What was that?
+[.discourse-part]
+_William Leddra:_ What was that?
 
+[.discourse-part]
 The court answered, that he had owned, that those who were put to death, namely,
 William Robinson, Marmaduke Stevenson, and Mary Dyer, were innocent +++[+++of that]
 for which they died; and that he would not put off his hat in court,
 and that he would say thee and thou to the magistrates.
 
-William Leddra: Then you put me to death for speaking English,
+[.discourse-part]
+_William Leddra:_ Then you put me to death for speaking English,
 and for not putting off my clothes.
 
 After some more words they pronounced sentence of death upon him,
```

Comment:
lets put a small-break between this comment of Leddra "putting of my clothes", and the beginning of the next paragraph "After some more words"...